plugins/uberdev/lib/review-fleet-args.sh

```
@@ -271,6 +286,274 @@ review_fleet_bind_persistence() {
+      --ci-authority-sha256 "$authority_sha256")" || return 2
+    row="$(jq -cn --arg edge "$edge" --argjson index "$index" --arg instance "$instance" \
+      --arg binding "$binding" --arg result "$dir/result.md" --arg status "$dir/status.json" \
+      '{edge:$edge,index:$index,instance:$instance,binding:$binding,result:$result,status:$status}')" || return 2
+    printf '%s\n' "$row" >>"$ledger" || return 2
+    if [ -z "$REVIEW_FLEET_NONCE_POOL" ]; then
+      REVIEW_FLEET_NONCE_POOL="$nonce"
+    else
+      REVIEW_FLEET_NONCE_POOL="$REVIEW_FLEET_NONCE_POOL,$nonce"
+    fi
+  done <"$authority_ledger"
+  REVIEW_FLEET_CONFLICT_COUNT="$index"
```

Shellcheck (0.11.0)

[warning] REVIEW_FLEET_CONFLICT_COUNT appears unused. Verify use (or export if used externally).

(SC2034)